### DISCUSSION TITLE
組版部分の構成

### DISCUSSION THREAD

**Discussion by kidayasuo** [w3c/jlreq-d#63] [editor-mtg]
# 2024-9-5 編集会議の結果

出席：敏先生、山口さん

章立てに変更を加えた
- 章の深さを浅くした。現在のドラフトでは3章で組版のほぼ全てをカバーしているが、階層を浅くして、3章-11章に展開した。これによって、内容を俯瞰しやすくなるし、Script Resource Pageからのリンクや、機能から引く辞書的な利用もやりやすくなると期待する。
- 現在 3.1 の用語の定義はどこかに独立させる
- 組版の詳細に入る前に、基礎的な仕組み、アーキテクチャを説明する。和字の組版のモデルとしてボディを並べる方法を説明する。次にベースラインがある場合にそれをどのように適用するのか、という現実的な話をする。その後の章で組版の詳細に入って行く。
- 見出しの例にあるように、あると期待されるトピックに関して書くことがない場合、1) 項目をなくしてしまう、2) 項目があるけれど、デジタルだからいらないとか、日本語独自のものがない、などと書く二通りの方法があるが、後者の方法を取る。項目が無い場合、読者にとってはなぜ無いのか分からなくなる。山口さんの良い指摘。
- まだ完全に満足していないところもあるが、とりあえずこれで構成してみる

----
## 3. 日本語組版の基本
> 詳細ではなくアーキテクチャの話

### 3.1 ボディベースの組版をまず説明
> ここではまず和字の基本を説明して、和字の組版は欧文ベースラインとは異なるシステムであることを説明。和字の組版の基本をわかってもらうためにある。本格的に説明すると言うよりも、ボディによる組み方のアーキテクチャ（中心揃え＋ベタ）を簡単に説明する。ベースラインという概念はないよ。サイズの異なる文字の配列について、中央揃えだと説明。
#### 字詰め方向（X方向）
- 和文組版のベタ組みが基本（約物も同じ）
- どこで分割してもいいよ
#### 行送り方向（Y方向）
- ボディが基本（例えば、文字サイズが異なれば中心で揃える。ベースラインという概念がない）

### 3.2 ベースラインシステムでの組版を説明
>しかし早い目にベースラインの中にどう組み込むかの現実的な話に持って行く。なぜなら、それがエンジニアが実際に対処しなければならない状況だから。混植は別の話。

- ベースラインシステムでの和字の配列

## 4. 行への文字の配置
> プログラムで行の組版処理を組むときに考えなければならい問題はここ

基本はボディをベタだけれど例外があるよ。例外の処理が腕の見せ所だよとアジ演説をする。それを以降で説明するよ。
本文プロポーショナル組の存在について一言。

## 7. 縦組
- 組方向の変更
- 縦組と横組で字形等が異なる例
- 数字の表記と組版処理
- ラテン文字等の処理
- 縦組と横組の句読点
- 縦組と横組の括弧類
- 縦組と横組で注意が必要な行処理等
- 縦組と横組で異なる注等の配置処理

## 8. 段落の処理（複数の行の処理）
- 行揃えの選択
- 段落の示し方
- 行長
- 行間
- 箇条書き

## 9 見出し
> 書くことあまりないけど、なければ、日本語独自のものがない、と書く。

### 11. 2 文字サイズに関わる問題
- 大きいサイズの文字の処理（ここに詰め組も来る）
- 小さいサイズの文字の処理（書くことある？）
- オプティカルサイジング（サイズに適した書体）

**Comment by kidayasuo** [comment]
ミーティングの後、時間が経っての感想：
- ？「3. 日本語組版の基本」、は、はっきり内容を表して「3. 日本語組版のアーキテクチャ」が良いかもしれない
- ○「5. ルビ」は行の分割に関わるし、行への文字の配置のプログラムの中で考慮しなければならないので、現在の位置でよし
- ？「6. 行の分割」と「7. 縦組」は逆でも？
- ×「9. 見出し」がどうも収まりが悪い
- ○「10. 注」は高度な構造なので、「8. 段落」よりも後、なのでそこでよし
- ○「11. 読みやすさと視覚効果」はある意味両方とも視覚的な調整なので、収まりよし

**Reply by himorin** [reply]
どちらかというと7.縦組みを4.行への文字の配置の直後に持ってくる方が「配置」という点でのまとまりではいいのかなという気はします。ルビも分割も行から少し離れる配置ですが、縦組みの行の中の文字の扱いは行の中身そのものですし。
見出しの収まりですが、ページから離れた中で、段落からページもしくは文章全体へつなぐ間という概念が、基本版面がなくなったことで自由になった、ところからのすわりの悪さなのかなという気はしています。JLreq-dの議論の中で図表とそれに付随するタイトル・文章の関連性などの議論も一部ありましたが、そういった「段落」でない何かをどう配置するかという一項目を置くというのも一つの手かなと思ったりもするのですが、いかがでしょう？（そういう意味では割注とかの本文に押し込まれる何かでない、脚注とかもこの分類に入るかもしれませんが）

**Comment by KobayashiToshi** [comment]
>ミーティングの後、時間が経っての感想： 

> - ？「3. 日本語組版の基本」、は、はっきり内容を表して「3. 日本語組版のアーキ
>テクチャ」が良いかもしれない 

いいかも，でも，内容がまとまってから，また考えよう．

> - ？「6. 行の分割」と「7. 縦組」は逆でも？ 

“6. 行の分割”は，基本的な事項，“7. 縦組”は，いってみれば，補足的事項なので，現行通りでいいのでは？

> - ×「9. 見出し」がどうも収まりが悪い 

そう思う，まあ，しばらくほっておこう．そのうちに“10. 注”と含め，何か考えられるかもしれない．

**Comment by KobayashiToshi** [comment]
X方向とY方向の文字の配置とベースラインシステムの説明ですが，以下のようなことは指摘できるかもしれません．

まず，用語ですが，“出版編集技術　第二版　下巻”の32ページに“寄り引き”と“上がり下がり”という用語の説明があります．縦組でも，横組でも行に文字を並べた場合，左右又は上下で文字位置にずれがないかどうかを言う用語です．

そして，母型の製作では，この“寄り引き”と“上がり下がり”といことが問題とされ，それがでないように慎重に母型は製作されるということです．活字は，ある意味で改定作業がデジタルと比べ，容易で，日常の校正作業，その他において，“寄り引き”と“上がり下がり”は問題とされることがあり，それに応じて母型の手直しも行われていたのです．つまり文字は日常的に鍛えられていたのです．

文字が並べられた場合，左右又は上下のずれがないかどうかは，慎重に検討され，そうしたことがでないように字形の設計やボディに対する字面の位置は，決められていたのです．ですのでＸ方向ではボディを密着させ，Ｙ方向では，ボディの中心をそろえて文字を配置すれば（同一サイズでは，ボディの端もそろう），左右又は上下にずれがでない文字配置が可能になるということです．

この場合，あくまで視覚的に見て左右又は上下のずれがでないようにするので，字面は，必ずしもボディの天地左右中央に配置することにはならない場合もあるということです．

そして，和字ではボディのどこに字面を配置すればいいかが検討されるのであって，ある物理的な位置，例えばラテン文字におけるベースラインという基準は，和字では，その考え方にはない，ということです．

デジタルな世界でも，活字起源のフォントがありますが，それだけではなく，こうした活字の設計の考え方，ボディの中に，“寄り引き”と“上がり下がり”がでないように文字は設計されるという考え方は，いまでも同じであるといえるでしょう．

ですので，結論としては，以下のようになる．

文字を行に配置する場合，縦組おいても横組おいても，視覚的に見て左文字位置が左右または上下にすればでないように配置される必要がある．和字の場合，これは字面の設計，及び字面のボディへ配置位置を慎重にデザインすることで実現している．この場合，あくまで視覚的に見て左右又は上下のずれがでないようにするので，字面は，必ずしもボディの天地左右中央に配置することにはならない場合もでてくる．なお，ラテン文字の場合，Y方向はベースラインという基準線を設け，文字をデザインする．しかし，和字の場合，どこに字面をボディのどこに配置すればいいかが検討されるのであって，ラテン文字におけるベースラインという基準線は和字のデザインでは設けいていない．

したがって，和字を行に配置する場合，特別な事情がない限り，Ｘ方向ではボディを密着させ，Ｙ方向では，ボディの中心をそろえて文字を配置すれば（同一サイズでは，ボディの端もそろう），左右又は上下にずれがでない文字配置が可能になる．

**Reply by kidayasuo** [reply]
なんとなくはわかるのですが、ここで「ずれ」は正確に何を意味しますか？　これが定義されていないので、「ずれがでない文字配置」が正確に何を意味するのかわかりませんでした。

説明の分量として、上はちょっと長すぎるように思いました。必要な情報を含みつつできるだけ簡潔にしたいです。フォントがちゃんと精密に作ってあることを前提とすると、ボディをその中央を基準に密着して並べる、と書くのでは不十分でしょうか？

**Comment by KobayashiToshi** [comment]
“ずれ”とは，“寄り引き”という言葉にあるように，文字を行に配置した場合に，左右のどちらかに寄って見えることです．“上がり下がり”は，上下に上がりすぎ，または下がりすぎに見えることです．

以下ではどうですか．

文字を行に配置する場合，縦組おいても横組おいても，視覚的に見て文字が左右または上下のどちらかに寄って見えないように配置される必要がある．和字の場合，これは字面の設計，及び字面のボディへの配置位置を慎重にデザインすることで実現している．なお，ラテン文字の場合，Y方向はベースラインという基準線を設け，文字をデザインするが，和字にはベースラインという基準線は設けていない．

したがって，和字を行に配置する場合，特別な事情がない限り，ボディをその中央を基準に密着して並べていけばよいことになる．

**Reply by kidayasuo** [reply]
視覚的に左右や上下に寄って見えない=視覚的に中心に揃って見える、という理解で良いでしょうか？

「これは字面の設計，及び字面のボディへの配置位置を慎重にデザインすることで実現している」はデジタル的な言葉で言うと「日本語書体のグリフはボディの中に各々のグリフが中央に見えるようにデザインされている」でしょうか。

日本語書体をデザインする際に重要な考慮点はこれ以外にも字間などいろいろあるとは思いますが、この部分はちゃんとした日本語書体があるという前提でテキストシステムを書く人や、そのフォントを用いてデザインする人が対象ですから、軽くで良いかと思います。ちゃんとした日本語書体を前提とすると、ボディを隙間なく並べること、またボディを隙間なく並べることを前提としてデザインされている、との知識があれば良いかと思います。日本語の文字のデザインのうち、ここで述べられた、グリフが中心に揃えてデザインされている、という点はここで重要ですね。

「したがって」は具体的にどこを指して、その当然の帰結として、と述べているのか、特にラテン文字の議論が挟まっていることもあって分かりにくくなっていると思います。また、サイズが同じ限り、（グリフがきちんとデザインされていれば）ボディのどこを基準にとっても同じですから、「中央を基準に」は異なるサイズの文字を一行に配列する場合にのみ明らかになるのだと思います。

異なるサイズの文字を一行に配置する場合に中央揃えにするということは、各々の文字が中央に揃えてデザインされていることからも、自然な帰結となりますね。

**Comment by KobayashiToshi** [comment]
> 視覚的に左右や上下に寄って見えない=視覚的に中心に揃って見える、という理解で良いでしょうか？

うーん，そうですが，ちょっと違っているようにも思います．現在でも，読んでいると，文字位置がずれているケースをたまに感じる場合があります．まさに位置がずれて見えるのであって，縦組でいえば，文字が右か左に寄って見えるのです．文字の中心を認識して，その中心がずれているという感覚ではなく，文字が右か左に寄って見えるのです．ですから，このずれを“寄り引き”というのです．そして，この“寄り引き”というのは，前後に配置される文字とのバランスでもあるので，単に中心をそろえるのではなく，テスト組で問題箇所を眼でみて探り，微妙に位置を直す必要があるのです．活字組版は，それが日常の作業としても行われ，修正していった．そうした経験が現在にも続いているのです．

ここは，ベースラインシステムと違いをというのが前回の木田さんの話にあったので書いたのですが，ここは微妙な問題ですので，こうしたことを含め，次回の編集会議で議論しましょう．

まず，行への文字の配置は，通常はベタ組（ボディを密着させる）にしていけばいい，という箇所で，何をいうかという問題です．最も簡単には，一般に，ベタ組を前提にフォントは設計されているので，ベタ組でいいといえば済む話でもある．

そして，実は，ベタ組（ボディを密着させる）にしないケースが日本語組版では出てくる．これを如何にうまく処理するかが問題である，という話につなげたい．その前提に何をいうかという問題かと思います．

**Reply by kidayasuo** [reply]
なるほど、結果的的には中央揃えと表現しても間違いではないように思いますが、感覚的に捉えられるのは上下左右方向へのズレということですね。

同じ文字でも、前後の文字によって位置の調整が必要になりますか？　つまり活字ができた後、組む段になって、その場その場で左右の調整を入れていたかということです。

デジタルフォントの文字も、周りの文字を色々に変えて、どの文字ともバランスよくなるように調整を繰り返して完成に近づけてゆきます。

ベタ組みにしないケースについては、以降の節で、約物、あけ組、詰め組、などボリュームのある説明をしてゆきますので、「ボディを密着させる組み方が基本だが、その例外となる処理が日本語組版の要である。それについて以降の節で解説する」というようなサラッとした言い方ではどうでしょう？

**Reply by kidayasuo** [reply]
編集会議しましょう。メーリングリストに出しますね。

**Comment by KobayashiToshi** [comment]
> どちらかというと7.縦組みを4.行への文字の配置の直後に持ってくる方が「配置」という点でのまとまりではいいのかなという気はします。ルビも分割も行から少し離れる配置ですが、縦組みの行の中の文字の扱いは行の中身そのものですし。

ここは，行への文字の配置という問題に限られていなく，約物の配置ではなく，その使い方から段落処理に関連した事項も含まれているので，とりあえず，現状でいいのではないかな．ここの内容も，今後変わる可能性もあり，このような考え方のあることをふまえ，今は，とりあえず現状のままとして，あとで，また，考えましょう．

**Comment by KobayashiToshi** [comment]
> なるほど、結果的的には中央揃えと表現しても間違いではないように思いますが、感覚的に捉えられるのは上下左右方向へのズレということですね。

そういうことです．別に表現すれば，ある種の漢字は，文字の中心が認識できるが，特に仮名の中心となると，とらえどころがない．でも仮名が並んでいる場合，ある文字が，どちらかに寄っているな，ということは感覚として認識できる．

読んでいて，あれ，ちょっとずれているな，と現在でも，たまに感じる場合があり，それは，読書の妨げになることもあります．

> 同じ文字でも、前後の文字によって位置の調整が必要になりますか？

あります．ですから，いろいろと検証しないといけない．それは難しいことだと思います．それに，デジタルでは，発売前に徹底的にやるのでしょうが，それ以後は，改訂という機会でないとがむつかしいのではないか．それに対し，活字では，ユーザー（出版社）からも日常的に指摘され，あるいは，印刷所によっては，暇があるとテスト組でながめてみるということも行っていたようです．また，その改訂も比較的に（そう頻繁ではないと思いますが），ある意味で日常的にできる環境にあったように思います．

> つまり活字ができた後、組む段になって、その場その場で左右の調整を入れていたかということです。

それはありません．

**Comment by KobayashiToshi** [comment]
> ベタ組みにしないケースについては、以降の節で、約物、あけ組、詰め組、などボリュームのある説明をしてゆきますので、「ボディを密着させる組み方が基本だが、その例外となる処理が日本語組版の要である。それについて以降の節で解説する」というようなサラッとした言い方ではどうでしょう？

いいように思います．25日に相談しましょう．

例えば，第3章の詳細に入る前提で何を言うかということで，
-  ベタ組で，まあ，いいよ．
- どこで2行に分割してもよいよ．

実は，これだけの機能は，少し前のテキスト作成のアプリケーションである“エディタ”（私は“WZ”（今では開発はストップしている）を使用しているが，だいぶ前から，行頭禁則など処理してくれる）がそうであり，それしかできなかった．でも，読むという点では，読めた．これは原稿だからということかなあ．

しかし，読者に支障なく読んでもらうためには，そうでない処理が必要になる場合があり（詳しくは以下で述べる……），その処理を如何に上手に処理するかが，組版の評価に大きく影響する事項でもある，みたいに話をしていく．

25日（水）に，相談しましょう．

**Comment by himorin** [comment]
@KobayashiToshi @kidayasuo ここでの議論は、「実装編：日本語フォントの要件」に関連しそうだと思うのですが、まだ文章がないのでとりあえず章のメモに残しておきます
